<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouTube_Downloader</class>
 <widget class="QMainWindow" name="YouTube_Downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>908</height>
   </rect>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>911</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Download menu</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>891</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>431</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter video URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>431</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Use browse button for save location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>511</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>145</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>331</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <zorder>lineEdit_3</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>progressBar_2</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>label</zorder>
       <zorder>comboBox</zorder>
       <zorder>pushButton_5</zorder>
       <zorder>lineEdit_4</zorder>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Full Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>471</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter playlist URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>471</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse Download Location</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>60</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>110</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>340</y>
          <width>211</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>290</y>
          <width>581</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>230</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Video:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>232</y>
          <width>64</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>230</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Full Playlist Video:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>230</y>
          <width>64</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
